### PR TITLE
Use threshold-based footer animation for Android

### PR DESCRIPTION
**Pull request by samholmes**
### CHANGELOG

Does this branch warrant an entry to the CHANGELOG?

- [ ] Yes
- [x] No

### Dependencies

 none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [x] Tested on iOS device
- [x] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206481494003972

### CODE REVIEW

File: src/state/SceneFooterState.tsx [hunk 1]
```diff
@@ -161,7 +163,7 @@ export const FooterAccordionEventService = () => {
 
   useSharedEvent(scrollEndEvent, () => {
     'worklet'
-    if (Platform.OS === 'android') {
+    if (isAndroid) {
```
Comment (paullinator) [inline src/state/SceneFooterState.tsx:166]: Not related to this review, but it is a bit suspect that you have to do a `runOnJs` here for Android. This implies you'll always cross-the-bridge every time the user's finger lifts off the screen on scroll.

**Review by paullinator — APPROVED** [review]
Approved, but I'd like a discussion on the two points below.

File: src/state/SceneFooterState.tsx [hunk 2]
```diff
@@ -172,14 +174,14 @@ export const FooterAccordionEventService = () => {
 
   useSharedEvent(scrollMomentumBeginEvent, nativeEvent => {
     'worklet'
-    if (Platform.OS === 'android') return
+    if (isAndroid) return
```
[the remark this answers is not in the document]
Comment (samholmes) [reply src/state/SceneFooterState.tsx:177]: You're proposing to remove momentum scrolling for iOS?

File: src/state/SceneFooterState.tsx (hunk 1 above)
Comment (samholmes) [reply src/state/SceneFooterState.tsx:166]: The runOnJS is because there is no `setTimeout` on the UI worklet thread.

Comment (paullinator) [reply src/state/SceneFooterState.tsx:166]: I may need a recap on the advantage of the setTimeout. What happens if we remove it? Can you try and see if perf improves

**Comment by samholmes** [comment]
Deprecated for https://github.com/EdgeApp/edge-react-gui/pull/4804

**Pull request closed without merging**